.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,9 @@
+Resolves #
+
+
+
+#### Pre-Merge Checklist:
+- [ ] Code is documented in doxygen format
+- [ ] Code has automated tests
+- [ ] Zero compiler warnings
+- [ ] Formatted with `clang-format`
```

Comment:
Fair. When you go to the Travis page though on the build, it says that it failed because of clang.